dev/run

```diff
@@ -475,7 +475,6 @@ def boot_nouveau(ctx):
         "server",
         "nouveau.yaml",
     ]
-
```

Comment:
Was not this an unintended change?  If not, why?

Reply:
It just showed as a diff between the 3.3.x branch and I thought'd bring it over to minimize differences.

Reply:
Ah, now it makes sense.